examples/dao/test/dao-app-flow.js

```diff
@@ -0,0 +1,457 @@
+const { getProgram, convert } = require('@algo-builder/algob');
+const { Runtime, AccountStore } = require('@algo-builder/runtime');
+const { types, parsing } = require('@algo-builder/web');
+const { assert } = require('chai');
+const { ProposalType, Vote } = require('../scripts/run/common/common');
+const {
+  now, getAddProposalTx, getDepositVoteTokenTx, getWithdrawVoteDepositTx,
+  getClearVoteRecordTx, getClearProposalTx
+} = require('../scripts/run/common/tx-params');
+
+const minBalance = 10e6; // 10 ALGO's
+const initialBalance = 200e6;
+
+/**
+ * Test for scripts flow. Steps:
+ * + create DAO (setup)
+ *    1. Create Gov Token (ASA)
+ *    2. Create DAO App
+ *    3. Compile & fund lsig's (deposit, daoFund, proposal)
+ *    4. Add depositLsig address to DAO app
```

Comment:
please add information:
+ we test token voting
+ specify who is going to be a token holder

Reply:
ok